[RadioMelodieBridge] Fix to use the new website layout

sysadminstory opened the pull request:
The website layout has been slightly updated, so the bridge needed to be adapted.

sysadminstory committed:
[RadioMelodieBridge] Fix to use the new website layout

The website layout has been slightly updated, so the bridge needed to be adapted.

dvikan reviewed (commented):
Tested locally and the bridge is repaired. However the parsing of the publish date is broken. Im getting:
```
Notice: Trying to access array offset on value of type bool in /home/rssbridge/public/bridges/RadioMelodieBridge.php on line 69
```
and the offending code is:
```
preg_match('/([a-z]{4,10} [0-9]{1,2} [a-z]{4,10} [0-9]{4} à [0-9]{2}:[0-9]{2})/ms', $dateHTML, $matches);
$dateText = $matches[1];
$dateArray = strptime($dateText, $dateFormat);
```

https://www.php.net/manual/en/function.strptime.php:


Warning

This function has been DEPRECATED as of PHP 8.1.0. Relying on this function is highly discouraged.


The regex is working fine but the `strptime()` does not like the input and returns false.

sysadminstory committed:
[RadioMelodieBridge] Fix to use the new website layout

Remove the function strptime which will is deprecated in PHP 8.1 and
replace it with the function date_parse_from_format

sysadminstory committed:
[RadioMelodieBridge] Fix to use the new website layout

Coding style policy fixes !

dvikan commented:
I tested your change and now it works perfectly. Thank you @sysadminstory

sysadminstory commented:
As `strptime()` is deprecated, I replaced it with a new function using `date_parse_from_format()` instead !

sysadminstory committed:
[RadioMelodieBridge] Fix to use the new website layout

Two date issues fixed !

The regular eexpression did not match days with one digit (because the
day is always printed with two caracters, with a leading space), and did
not match months with accented characters like 'décembre'.

The date 'translate' function had a wrong placed duplicate for
'november' in the '$replace_en' array, which leaded to a wrong date conversion.
